src/Nova/Resource.php

```diff
@@ -4,11 +4,94 @@
 
 namespace Tipoff\Support\Nova;
 
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\BelongsTo;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Fields\Field;
+use Laravel\Nova\Fields\ID;
 use Laravel\Nova\Http\Requests\NovaRequest;
 use Laravel\Nova\Resource as NovaResource;
 
 abstract class Resource extends NovaResource
 {
+    protected array $filterClassList = [];
+
+    protected array $cardClassList = [];
+
+    protected array $lensClassList = [];
+
+    protected array $actionClassList = [];
```

Comment:
Allow resources to declare filters, cards, lenses and actions that might be loaded in other classes by class name.